src/Sepes.Tests/Services/Infrastructure/Study/StudyCreateServiceShould.cs

```diff
@@ -35,7 +29,7 @@ public async Task CreatingStudyWithoutNameShouldFail(string name)
         [Theory]
         [InlineData(null)]
         [InlineData("")]
-        public async void CreatingStudyWithoutVendorShouldFail(string vendor)
+        public async void FailIfStudyIsMissingVendor(string vendor)
```

Comment:
Did you change naming convention or is this just so it is shorter?

Reply:
Something i picked up at the TDD course, you see the class is named StudyCreateServiceShould, and then the Method name starts with Fail.., so the class name + test name reads as a sentence "StudyCreateServiceShould FailIfStudyIsMissingVendor"

Reply:
Ahh, that is clever!